server/hap2/hatohol/zabbixapi.py

```diff
@@ -311,7 +311,7 @@ def get_event_end_id(self, is_first=False):
         if not self.result:
             return
 
-        return int(res_dict[0]["eventid"])
+        return res_dict[0]["eventid"]
```

Comment:
The problem happen at other code.
So, I think to fix it collectively. I will make a Issue.